modules/runner/src/integrationTest/java/org/apache/ignite/internal/compute/ItLogicalTopologyTest.java

```diff
@@ -120,6 +137,46 @@ void receivesLogicalTopologyEvents() throws Exception {
         assertThat(events, is(empty()));
     }
 
+    @Test
+    void receivesLogicalTopologyEventsWithNodeAttributes() throws Exception {
```

Comment:
Do we have similar test for logicalTopologyOnLeader?

Reply:
Added, thanks!